### PR TITLE
Community - INFODEV-3689: Use WinSCP to transfer files

### PR DESCRIPTION
**Pull request by chri2547**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-ffa8e719ef/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e952da936e/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-51f3fa3f7b/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-7c7d892f70/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-ae03d8a434/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c45477d347/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-dac133f9f2/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-e9f10ff7ef/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-51b5b65738/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-12e784b3da/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c864463c3b/how-to/).

**Comment by cyrichardson** [comment]
@chri2547 On it.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c03fce9628/how-to/)

**Comment by cyrichardson** [comment]
@chri2547 Suggestions here: 
[transfer-files-using-WinSCP_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2957577/transfer-files-using-WinSCP_cyrEdits.docx)

**Comment by cyrichardson** [comment]
@chri2547 Might want to change 

created_by
--
Chris Moyer
 to

created_by
--
Rackspace Community

I think that's what folks have been using for other articles created from Community posts.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a0b2cca392/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-4c860ad3c4/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9d5fb62994/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-95a33ec5e1/how-to/).

File: content/cloud-servers/transfer-files-using-WinSCP.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+WinSCP is an open-source, free Secure File Transfer Protocol (sftp) and File Transfer Protocol (ftp) client for Microsoft® Windows®. You can use WinSCP to securely transfer files between your local Microsoft Windows machine and a remote server.
+
+This article describes how to download, install, and use WinSCP to transfer files between computers. This article also describes how to create files and folders, and set permissions..
+
+### Download and install WinSCP
+
+1. [Download](https://winscp.net/eng/download.php) WinSCP.
+
+2. Double-click WinSCP to begin the installation process.
```
Comment (catlook) [inline content/cloud-servers/transfer-files-using-WinSCP.md]: maybe bold WinSCP?  or say double-click the WinSCP application or executable or something?

File: content/cloud-servers/transfer-files-using-WinSCP.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+3. Select **Typical** installation or **Custom** installation for the Setup type and click **Next**.
```
Comment (catlook) [inline content/cloud-servers/transfer-files-using-WinSCP.md]: bold Setup

File: content/cloud-servers/transfer-files-using-WinSCP.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+### Transfer files
+
+1. Launch WinSCP.
+
+2. To transfer files, drag and drop files from your local machine that is shown in the left pane, to the server file system that is shown in the right pane.
+
+    If you don't have permission to write to the server file system, an **Error** dialog box appears.
+
+    **Note**: The lock on the lower right side of the **Commander** window indicates that the connection is secure. If you double-click the icon, you see the security features of your connection, such as the encryption algorithm used by sftp.
```
Comment (catlook) [inline content/cloud-servers/transfer-files-using-WinSCP.md]: put back ticks around sftp

File: content/cloud-servers/transfer-files-using-WinSCP.md (hunk 1 above)
Comment (chri2547) [reply content/cloud-servers/transfer-files-using-WinSCP.md]: Done. changed to double-click the WinSCP executable...

File: content/cloud-servers/transfer-files-using-WinSCP.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+created_by: Rackspace Community
+last_modified_date: '2019-03-12'
+last_modified_by: Chris Moyer
+product: Cloud Servers
+product_url: cloud-servers
+---
+
+WinSCP is an open-source, free Secure File Transfer Protocol (sftp) and File Transfer Protocol (ftp) client for Microsoft® Windows®. You can use WinSCP to securely transfer files between your local Microsoft Windows machine and a remote server.
+
+This article describes how to download, install, and use WinSCP to transfer files between computers. This article also describes how to create files and folders, and set permissions..
```
[the remark this answers is not in the document]
Comment (chri2547) [reply content/cloud-servers/transfer-files-using-WinSCP.md]: Removed.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3964df4a4d/how-to/)

File: content/cloud-servers/transfer-files-using-WinSCP.md (hunk 3 above)
Comment (chri2547) [reply content/cloud-servers/transfer-files-using-WinSCP.md]: Done.

**Pull request merged**